db/maintainer.js

```diff
@@ -13,23 +13,101 @@ class MaintainerDbController {
     return pkgs.map(({ _id: id, ...rest }) => ({ id, ...rest }))
   }
 
-  async getRevenue ({ maintainerId }) {
-    const packages = await this.db.collection('packages').find({
-      maintainers: { $elemMatch: { maintainerId } }
-    }).toArray()
-    return packages.reduce((totalRevenue, pkg) => {
-      const { revenuePercent } = pkg.maintainers.find((maintainer) => maintainer.maintainerId === maintainerId)
-      return totalRevenue + (pkg.totalRevenue * (revenuePercent / 100))
-    }, 0)
-  }
-
   async updateIlpPointer ({ userId, ilpPointer }) {
     return this.db.collection('users').updateOne({
       _id: ObjectId(userId)
     }, {
       $set: { 'payoutInfo.ilpPointer': ilpPointer }
     })
   }
+
+  // @returns an array of [{ _id: <packageId>, payout: <payoutInMS> }]
+  async getPendingPayout ({ maintainerId }) {
```

Comment:
as far as I can tell, this returns the money we owe the specified maintainer, per package, that we _haven't tried sending them yet_... is that correct? (that is, it doesn't include stuff from that new "other" ledger on the package where we keep track of pending payouts by maintainer). if that's correct... this is getting confusing 😄 but shouldn't it include that? (since it's still pending from their point of view).

Reply:
correct haha and yes but we can add a TODO later to add that since we haven't designed it